### DISCUSSION TITLE
Welcome to ESP32-Flight-controller- Discussions!

### DISCUSSION THREAD

**Discussion by pratikPhadte** [pratikPhadte/ESP32-Flight-controller-#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Aditya6511** [comment]
Hey Pratik!
First, I’d like to thank you for sharing your amazing work on building a drone. Your project has been a huge inspiration, and I truly appreciate the effort you’ve put into making the repository available for others to learn from.

I’m currently working on a similar drone project, but with a slight twist. I aim to replace the RF module and physical controller with the ESP32’s built-in Wi-Fi and Bluetooth capabilities, enabling control via a mobile phone application. I believe this approach could modernize the control mechanism while leveraging the ESP32’s versatile features.

I would greatly appreciate your guidance regarding **Code Modifications** and **Circuit Adjustments**.

If you have insights, resources, or advice on handling real-time control responsiveness via WiFi/Bluetooth, that would also be incredibly helpful.

**Comment by ruggis58** [comment]
Hi Patrick
I followed your video on the flight controller with esp32 with great interest and I wanted to ask you a couple of things:
1- have you considered using ESP-NOW? I'm making a transmitter with esp-now for a rover.
2- I wanted to make a model of the avatar scorpion that has two propellers. do you think it's possible to use your project to make a controller for two propellers?
3- where can I find the banana PCB?
thanks for your attention and good work.

**Comment by Dattasai-777** [comment]
HI Pratik 
I'm datta sai aerospace engineer, drone enthusiast.
 im having a done computation next week . 
i want your support urgently to make the drone using teensy 4.0 or esp 32. I bought all required components to make it frame , mpu 6050 , bmp 280, teensy 4.0, esp 32 , transmitter , receiver, lipo battery.
im having zero knowledge of coding struggling to program.
need your help in urgent .
  pls guide me in programing & calibrating the drone.
pls reply as soon as you see this i'm running out of time.

**Comment by subham7272** [comment]
Hey Pratik Sir 
I Trying To Building Pcb Using Your Scamatics But Is Not Work I Thik You Use More Components In Pcb 
And Why Are You Not  Useing Comapas In Your Pcb After Desing,  Comapas Plase Is Empty

And How Can I Build Esp32 Transmitter And Reciver Can You Make a Video In This Topic
